test/integration/costumes.test.js

```diff
@@ -85,6 +85,17 @@ describe('Working with costumes', () => {
         await clickText('costume1', scope.costumesTab);
         await clickText('Convert to Vector', scope.costumesTab);
```

Comment:
It looks like something got screwed up here, the new test stole the bottom of this one?